search-app/src/cmr/search/services/acl_service.clj

```diff
@@ -23,15 +23,36 @@
   [context acls concept]
   true)
 
+;; When plans solidify around acl checks for service concepts, we'll update this.
```

Comment:
Needs a ticket number. If we don't have a ticket number for ACLs for services we need to write one.

Reply:
Yeah, Liu and I looked at this comment this morning (it's a copy-paste) and where were wondering where it came from and what it might be referring to. I think I may have actually written the original in the variables version of this sys-int test ... I'll dig and try to remember, and then update the comment.

Reply:
I'm having vague memories of the comment origins; I think I made them after a pairing session with Liu (the first time we ran into the testing issues as a result of this code not being updated when variable concepts were added). I think that comment came from me poorly paraphrasing something Liu said on the call, so I've updated it to essentially say the same thing that the tag comment says: basically, there's no ACLs for variables and services right now.

In other words, done ;-)